docs/local-development.md

```diff
@@ -529,7 +529,7 @@ We shallow render the component to a JSON and save it to a fixture. Subsequent r
 
 ```js
 it("should render a button", () => {
-  const onClick = jest.genMockFunction();
+  const onClick = jest.fn();
```

Comment:
`genMockFunction` was removed and was an undocumented API. Removing it from our examples and code.

https://github.com/facebook/jest/pull/6173